tutorials/local_build.md

```diff
@@ -33,9 +33,12 @@ cd openwhisk-runtime-python
 2. Build docker
 
 Build using Python 3.7 (recommended). This tutorial assumes you're building with python 3.7.
-
+Run local_build.sh with the correct parameters to build docker
```

Comment:
Hoping that you could help out and make this markdown more readable?

- Reduce the use of numbered lists (and do not start at zero 0.)
- Create more subsection headings to break up the numbered lists 
  - e.g., "Build the iamge", "Run the image", "Test the image" (with Curl) etc.
- Create subsections that show case "Testing with Helloworld" and explain init/run entry points into image.

May wish to align with work in the nodejs runtime's README and docker build/run/test README:
https://github.com/apache/openwhisk-runtime-nodejs/pull/227

Please know further updates to the files there are expected.